src/packages/spinner/ducks.js

```diff
@@ -57,39 +55,5 @@ export default combineReducers({
  * SELECTORS
  */
 const selectSpinner = state => state.spinner;
-const selectIsInProgress = (state, apiCallId) => !!selectSpinner(state).apiCalls[apiCallId];
-const selectGlobalCounter = state => !!selectSpinner(state).globalCounter;
-
-/**
- * DECORATORS
- */
-export function withApiCall(apiCallId, saga) {
```

Comment:
why removing this?

Reply:
Its not uses anywhere and i dont want to tight spinner pkg to saga. Anyone can create this helper in app code